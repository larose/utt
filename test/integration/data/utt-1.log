2014-03-14 08:00 hello
2014-03-14 09:00 hard work

2014-03-17 09:00 hello
2014-03-17 10:15 hard work

2014-03-19 09:00 hello
2014-03-19 12:00 asd: A-526
2014-03-19 13:00 lunch**
2014-03-19 14:00 hard work
2014-03-19 14:15 qwer: b-73
2014-03-19 14:30 asd: A-526
2014-03-19 14:45 qwer: C-123
2014-03-19 15:00 qwer: a-9
2014-03-19 16:00 black out ***
2014-03-19 16:30 A: z-8
