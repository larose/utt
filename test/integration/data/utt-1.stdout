
---------------------- Wednesday, Mar 19, 2014 (week 12) -----------------------

Working Time: 7h30 (5h30 + 2h00)
Break   Time: 1h00

----------------------------------- Projects -----------------------------------

(3h00)     : -- Current Activity --, hard work
(0h30) A   : z-8
(3h15) asd : A-526
(0h45) qwer: a-9, b-73, C-123

---------------------------------- Activities ----------------------------------

(2h00)     : -- Current Activity --
(1h00)     : hard work
(0h30) A   : z-8
(3h15) asd : A-526
(0h15) qwer: a-9
(0h15) qwer: b-73
(0h15) qwer: C-123

(1h00) : lunch**

----------------------------------- Details ------------------------------------

(3h00) 09:00-12:00 asd: A-526
(1h00) 12:00-13:00 lunch**
(1h00) 13:00-14:00 hard work
(0h15) 14:00-14:15 qwer: b-73
(0h15) 14:15-14:30 asd: A-526
(0h15) 14:30-14:45 qwer: C-123
(0h15) 14:45-15:00 qwer: a-9
(1h00) 15:00-16:00 black out ***
(0h30) 16:00-16:30 A: z-8
(2h00) 16:30-18:30 -- Current Activity --

